website/docs/core-concepts/components/terraform-providers.mdx

```diff
@@ -147,7 +153,35 @@ The generated `providers_override.tf.json` file would look like this:
 Terraform then uses the values in the generated `providers_override.tf.json` to
 [override](https://developer.hashicorp.com/terraform/language/files/override) the parameters for all the providers in the file.
 
+## `alias`: Multiple Provider Configuration in Atmos Manifests
+
+Atmos allows you to define multiple configurations for the same provider using a list of provider blocks and the
+`alias` meta-argument.
+
+The generated `providers_override.tf.json` file will have a list of provider configurations, and Terraform will use and
+override the providers as long as the aliased providers are defined in the Terraform component.
```

Comment:
We might want to note that since it's a list, it doesn't currently work with deep-merging of stacks, when used this way.